### PR TITLE
Update instructions 4jan17

### PR DESCRIPTION
**Pull request by capalmer85**

### CODE REVIEW

**Comment by capalmer85** [comment]
@DavidSticklandCMS @plujan I've reviewed, revised and reduced the instructions.  Please have a look and merge.  @plujan It is probably a good idea for you to describe the scripts you created with examples for usage in the same way in the README.me.

File: README.md [hunk 1]
```diff
@@ -1,45 +1,58 @@
...
+1. Check out the group's version of the tools
     `git clone https://github.com/CMS-LUMI-POG/Normtags`
 
-2. Step 1 clones only the master BRANCH of the CMS-LUMI-POG/Normtags (from now on origin). To fetch all the BRANCHES of the origin, 
-   `git fetch origin`
-   You can see the list of all BRANCHes with
-   `git branch -a`
+2. Now, if you want to further release the local changes, the following steps have to be done. Make a remote to your fork 
+   `git remote add YOURGITUSERNAME git@github.com:YOURGITUSERNAME/Normtags.git'
+   Make sure that your remote repositoty has been added
```
Comment (plujan) [inline README.md:37]: small typo here

**Review by plujan — APPROVED** [review]

**Comment by plujan** [comment]
Hi @capalmer85, I did document the scripts that I use for the online normtags in README_normtag. If you prefer we can merge these into a single README.

**Comment by capalmer85** [comment]
@plujan  Thanks!  I didn't see that file before.  The README.md shows up at the bottom of the main page (https://github.com/CMS-LUMI-POG/Normtags).  I think it is best to put it there.  I see that my edits didn't come out quite right so I'll make another PR to clean that up shortly.

**Comment by plujan** [comment]
@capalmer85 Sure, if you'd like to merge the documentation in that PR too, that would be great.

**Comment by capalmer85** [comment]
Will do.

**Pull request merged**